src/screens/ImportWallet/ImportWallet.js

```diff
@@ -144,14 +144,14 @@ const ButtonInner = styled.View`
 `;
 
 const ButtonLabel = styled(MediumText)`
-  color: ${props => props.blue ? baseColors.electricBlue : baseColors.white};
+  color: ${({ blue, theme }) => blue ? theme.colors.primary : theme.colors.control};
   font-size: ${fontSizes.big}px;
   margin-bottom: 2px;
 `;
 
 const ButtonIconArrow = styled(Icon)`
   font-size: ${fontSizes.big}px;
-  color: ${props => props.blue ? baseColors.electricBlue : baseColors.white};
+  color: ${({ blue, theme }) => blue ? theme.colors.primary : theme.colors.control};
```

Comment:
Just thinking here if `blue` prop is set we go for `primary` color which might change as well, maybe we should rename prop as `defaultColor` or `defaultTheme` or `primaryColor`?